Don't recognize directories as libraries.  eg. If we have system
libvirt and OCaml bindings in $OCAML_LIB_DIR/libvirt, then we don't
want '-lvirt' to "discover" the OCaml bindings directory.

Index: mingw-ocaml/build/flexdll/reloc.ml
===================================================================
--- mingw-ocaml.orig/build/flexdll/reloc.ml	2013-03-22 18:18:45.611942176 -0500
+++ mingw-ocaml/build/flexdll/reloc.ml	2013-03-22 18:18:55.000000000 -0500
@@ -109,7 +109,7 @@
   Filename.dirname (get_output1 (Printf.sprintf "@mingw_host@-gcc %s-print-libgcc-file-name" extra))
 
 let file_exists fn =
-  if Sys.file_exists fn then Some fn
+  if Sys.file_exists fn && not (Sys.is_directory fn) then Some fn
   else if !use_cygpath && Sys.file_exists (fn ^ ".lnk") then
     Some (get_output1 (Printf.sprintf "cygpath -m %s" fn))
   else None
